### PR TITLE
chore(cdk): `schematics` proprietary migrations

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [X] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?
Part of https://github.com/Tinkoff/taiga-ui/issues/1764

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/cdk/schematics/ng-update/constants/consts.ts [hunk 1]
```diff
@@ -122,4 +122,44 @@ export const CONSTS_TO_REPLACE: ReplacementConst[] = [
...
+            moduleSpecifier: '@taiga-ui/proprietary-banking',
+        },
+        to: {
+            name: 'TuiPreviewModule',
+            moduleSpecifier: '@taiga-ui/addon-preview',
+        },
+    },
+    {
+        from: {
+            name: 'TuiCarouselModule',
```
Comment (waterplea) [inline projects/cdk/schematics/ng-update/constants/consts.ts:147]: Proprietary Carousel is old and has different API. We should show a warning to check the new version docs.

File: projects/cdk/schematics/ng-update/constants/consts.ts [hunk 2]
```diff
@@ -122,4 +122,44 @@ export const CONSTS_TO_REPLACE: ReplacementConst[] = [
...
+            moduleSpecifier: '@taiga-ui/proprietary-core',
+        },
+        to: {
+            name: 'TuiCarouselModule',
+            moduleSpecifier: '@taiga-ui/kit',
+        },
+    },
+    {
+        from: {
+            name: 'TuiProgressModule',
```
Comment (waterplea) [inline projects/cdk/schematics/ng-update/constants/consts.ts:157]: Same here

File: projects/cdk/schematics/ng-update/constants/templates.ts [hunk 3]
```diff
@@ -278,4 +278,16 @@ export const TEMPLATE_COMMENTS = [
         comment:
             'See examples how create labels for ticks without this property (outside the component): https://taiga-ui.dev/components/range#segments',
     },
+    {
+        tag: 'tui-preview-host',
+        withAttr: 'ngProjectAs',
+        comment:
+            '"Preview"-component no more needs it and requires only import of TuiPreviewModule to the main module. See "Setup"-section: https://taiga-ui.dev/components/preview/Setup',
```
Comment (waterplea) [inline projects/cdk/schematics/ng-update/constants/templates.ts:285]: no longer*

**Review by waterplea — APPROVED** [review]

File: projects/cdk/schematics/ng-update/constants/templates.ts (hunk 3 above)
Comment (nsbarsukov) [reply projects/cdk/schematics/ng-update/constants/templates.ts]: fixed

File: projects/cdk/schematics/ng-update/constants/consts.ts (hunk 1 above)
Comment (nsbarsukov) [reply projects/cdk/schematics/ng-update/constants/consts.ts]: fixed

**Pull request merged**